### ISSUE TITLE
Show fingerprinting attempts in shields dropdown

### ISSUE THREAD

**Issue by TheCultLeader666** [brave/brave-browser#15446]
## Description 

Show some indication of when brave has randomized your canvas/webGL fingerprint, to let people easily identify why a site might be broken

## Steps to Reproduce

   1. Go to a website that uses some form of fingerprinting
   2. Go to the shields dropdown
   3. 

## Actual result:

It says 0 under the fingerprinting tab

## Expected result:
It will give a number of randomized fingerprints, and what was randomized

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave: 1.24.71 Chromium: 90.0.4430.72 (Official Build) beta (x86_64)

Revision: b6172ef8d07ef486489a4b11b66b2eaeed50d132-refs/branch-heads/4430@{#1233}

OS: macOS Version 11.3 (Build 20E5210c)

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
Haven't tried
- Can you reproduce this issue with the beta channel? 
Yes
- Can you reproduce this issue with the nightly channel? 
Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
N/A
- Does the issue resolve itself when disabling Brave Rewards? 
N/A
- Is the issue reproducible on the latest version of Chrome? 
N/A

**Comment by rebron** [comment]
cc: @pes10k @karenkliu

**Comment by pes10k** [comment]
Hi @TheCultLeader666 we used to report this but we stopped because there were way too many false positives for the signal to be useful.  We also didn't have a useful way of distinguishing benign uses of these APIs from malicious uses.  In general, it was a lot of information that we found users didn't understand (plus it made a number of sites / developers understandably upset to have their benign use of APIs labeled as "possibly privacy harming").

So, our current approach here is to pay a lot of attention to cases where default fingerprinting protections break sites, and either work with the site or tweak Brave default behavior to address compat problems there.  Not 100%, but in the vast majority of cases this has been a successful strategy. 

Also, going forward, we intend to keep using flags to control new brave features (including new fingerprinting protections) so that advanced users can use that to debug and change behavior.  We may add these to existing fingerprinting protections if / when we have to change their behavior (but probably won't add those flags otherwise).

Finally, we have (unscheduled) plans to build out the chromium devtools interface / network panel to show what is being blocked by network rules and similar (generally captured by [this issue](https://github.com/brave/brave-browser/issues/6458). If / when we get to that, we might also look into presenting farbling activity there too).

I'm going to close this in favor of https://github.com/brave/brave-browser/issues/6458 since thats our general approach here (for presenting non-typical-user level details about shields protections and activity).  BUt i think your concern is important so I will make a special note of this concern in that issue thread.  Thanks!
